release/nightly_tests/dataset/map_batches_benchmark.py

```diff
@@ -147,6 +147,58 @@ def run_map_batches_benchmark(benchmark: Benchmark):
                 num_calls=1,
             )
 
+    # Test map_batches whose output is large and will be split into multiple blocks.
+    # With the following configuration, the total data size is (num_output_blocks * 1GB).
```

Comment:
the total final dataset size is num_output_blocks * 1 GB right? the input dataset is smaller

Reply:
yes, I'll clarify this.